DEVGUIDE.md

```diff
@@ -23,15 +23,31 @@ The primary technical documents for the F# compiler code are
 
 ## Quick Start: Build, Test, Develop
 
-You can build the F# compiler for .NET Framework as follows:
+### F# Compiler (Linux)
```

Comment:
It mentions Linux but in before_install.sh the apt-get command are, I assume, ubuntu specific, maybe mention "tested with Ubuntu"?